style.css

```diff
@@ -12,6 +12,37 @@ html {
     text-align: center; 
 }
 
+#header-bg {
+    background-color: var(--background-color-header);
+    height: 80px;
+}
+
+#header {
+    margin: 0em 5em;
```

Comment:
could you use rem rather than em for better alignment and scaling to the rest of elements?

Reply:
please be sure to use rem in later version